### ISSUE TITLE
@harshibar when i run get_link.py it only opens glassdoor website , it doesnt search for jobs. plz help

### ISSUE THREAD

**Issue by aisha01malik** [harshibar/common-intern#14]
@harshibar @harshibar-youtube plz help

**Comment by eminvergil** [comment]
success = go_to_listings(driver) this function gives error

**Comment by Tuhin-thinks** [comment]
> success = go_to_listings(driver) this function gives error

Also, one may notice that he/she not getting relevant results from the search, because the search probably is working as location based, but the `position` text is never getting written.

I fixed the issue by adding some CSS selectors, the way of getting elements using fixed, `id` and `XPATH` isn't really effective.

Also, the page navigation is only of 4 pages, you can easily increase that and search as long as the search has results using the `page next` button.

**Comment by eminvergil** [comment]
@Tuhin-thinks yes i fixed that issue using **Full XPATH** but when it collects all **url** data it doesnt work,it doesnt fill the forms .Did you make this bot work ? and if so can i get contact with you or can you share your results? i would be appreciated.

**Comment by Tuhin-thinks** [comment]
Yes, apply job will not work according to the script.
It just works for `lever` and `greenhouse`.

But, what approach she's using is : to create automation rules for every website, but that's totally unrealistic!!
We cannot create selenium rules for all 100+ companies that are scrapped! So, I am thinking of a better solution and will come up with one in future when I will really need for this kind of script!

Also, for starters, get_links.py is insanely slow!!
Basically scrap 25-30 links in 15-16 minutes (just finds them, not scrap anything tho).
Lots of improvement needed, I won't open any new issue for this, just wait for existing ones to get fixed!
